R/helpers_ggbetween_t_subtitles.R

```diff
@@ -584,3 +584,226 @@ subtitle_t_bayes <- function(data,
+    Sigmad <- sqrt((n1 + n2)/(n1 * n2) + 0.5 * d^2/(n1 + n2))
+    Z <- -qt((1 - conf.level)/2, df)
+    lower.ci <- c(d - Z * Sigmad)
+    upper.ci <- c(d + Z * Sigmad)
+    method <- "Cohen's d"
+    tobject <- t.test(x, y, var.equal = TRUE, conf.level = conf.level)
+    tvalue <- tobject$statistic
+    dfvalue <- tobject$parameter
+    civalue <- attr(tobject$conf.int,"conf.level")
+    twosamples <- TRUE
+  }
+
+
+  # -------------- two paired samples in matching columns -------------------
+
+  if (length(formula) == 3 & isTRUE(paired)) {
+    if (is.factor(eval(formula[[3]], data))) {
+      outcome <- eval(formula[[2]], data)
+      group <- eval(formula[[3]], data)
+      group <- droplevels(group)
+      x <- split(outcome, group)
+      y <- x[[2]]
+      x <- x[[1]]
+    } else {
+      x <- eval(formula[[2]], data)
+      y <- eval(formula[[3]], data)
+      ind <- !is.na(x) & !is.na(y)
+      x <- x[ind]
+      y <- y[ind]
+    }#    return(x)
+    if (length(x) != length(y)) {
+      stop("paired samples requires samples of the same size")
+    }
+    n <- length(x)
+    df <- n - 1
+    r <- cor(x,y)
+    sd.est <- sd(x - y)
+    mean.diff <- mean(y) - mean(x)
+    d <- mean.diff/sd.est
+    Sigmad <- sqrt(((1 / n) + (d^2 / n)) * 2 * (1 - r)) # paired
+    Z <- -qt((1 - conf.level)/2, df)
+    lower.ci <- c(d - Z * Sigmad)
+    upper.ci <- c(d + Z * Sigmad)
+    method <- "Cohen's d"
+    diffscores <- as.vector(y - x)
+    tobject <- t.test(diffscores, mu = 0, var.equal = TRUE, conf.level = conf.level)
+    tvalue <- tobject$statistic
+    dfvalue <- tobject$parameter
+    civalue <- attr(tobject$conf.int,"conf.level")
+    twosamples <- FALSE
+  }
+
+  # -------------- apply hedges correction -------------------
+
+  if (hedges.correction == TRUE) {
+    method <- "Hedges's g"
+    d <- d * (n - 3)/(n - 2.25)
+  }
+
+  # -------------- calculate NCP intervals -------------------
+
+  if (isTRUE(noncentral)) {
+    st <- max(0.1, tvalue)
+    end1 <- tvalue
+    while (pt(q = tvalue, df = dfvalue, ncp = end1) > (1 - civalue) / 2) {
+      end1 <- end1 + st
+    }
+    ncp1 = uniroot(function(x)
+      (1 - civalue) / 2 - pt(q = tvalue,
+                             df = dfvalue,
+                             ncp = x),
+      c(2 * tvalue - end1, end1))$root
+    end2 <- tvalue
+    while (pt(q = tvalue, df = dfvalue, ncp = end2) < (1 + civalue) / 2) {
+      end2 <- end2 - st
+    }
+    ncp2 = uniroot(function(x)
+      (1 + civalue) / 2 - pt(q = tvalue,
+                             df = dfvalue,
+                             ncp = x),
+      c(end2, 2 * tvalue - end2))$root
+
+    if (isTRUE(twosamples)) {
+      ncp.upper.ci <- ncp1 * sqrt(1 / n1 + 1 / n2)
+      ncp.lower.ci <- ncp2 * sqrt(1 / n1 + 1 / n2)
+    } else {
+      ncp.upper.ci <- ncp1 / sqrt(dfvalue)
+      ncp.lower.ci <- ncp2 / sqrt(dfvalue)
+    }
+  }
+
+  # -------------- return results desired -------------------
+
+  if (isTRUE(noncentral)) {
```

Comment:
Everywhere in the package, I have tried to be consistent with the column names to follow the `tidyverse` style guide.

So can you change the column names here to-
`method`, `estimate`, `conf.low`, `conf.high`

The confidence interval column names are especially important since everywhere else (e.g., custom bootstrapping functions that output effect size CIs) we are using `conf.low` and `conf.high` and, when all effect sizes of interest are covered in the package, I plan to write a wrapper function and it will be convenient to have a consistent naming convention across outputs. 
(I am borrowing this convention from `broom` and `broom.mixed` packages.)

Reply:
Yes forgot that chnge is manual

Reply:
@ibecav This still needs to be resolved.

Reply:
This is a different issue, btw. 

Sorry about the awkward review format GitHub currently has.

Reply:
Done